### ISSUE TITLE
work environment problem

### ISSUE THREAD

**Issue by dahrs** [gabrielStanovsky/oie-benchmark#5]
Hello @gabrielStanovsky,
I have a problem trying when I try to launch the create_oie_corpus.sh  file. It returns : 
**IOError: [Errno 2] No such file or directory: './QASRL-full/newswire/propbank.dev.qa'**
According to your documentation the QASRL-full should only contain the QA-SRL corpus, which, downloaded from https://dada.cs.washington.edu/qasrl/#dataset looks like this:
**newswire_nosent.dev.qa
newswire_nosent.test.qa
newswire_nosent.train.qa
README.md
wiki1.dev.qa
wiki1.test.qa
wiki1.train.qa**
So did I miss something?

BTW Thanks for your paper and the benchmark

**Comment by gabrielStanovsky** [comment]
Hi @dahrs,
First, note that the newswire (=propbank) files do not contain the raw sentences, which are needed for the conversion as well (copyrights issues...).
As the QA-SRL site mentions, you need to get them separately from the PTB:
```
*The newswire data does not contain the original sentences. You will need to download and run the following python script with the CoNLL-2009 English training data to get the complete data.
```
Second, I assume that you're getting this error when running ```create_oie_corpus.sh```. Right?
The scripts currently expects the following folder structure:
```
QASRL-full/:
newswire/  README.md  wiki/

QASRL-full/newswire:
propbank.dev.qa  propbank.qa  propbank.test.qa  propbank.train.qa

QASRL-full/wiki:
wiki1.dev.qa  wiki1.qa  wiki1.test.qa  wiki1.train.qa
```
It seems that the way the QA-SRL corpus is being downloaded now has diverged from this structure.

There are two possible solutions:

1. Change the folder structure in your local setup to adhere to the one above, which the scripts expects.
2. Run ```qa_to_oie.py --in=INPUT_FILE --out=OUTPUT_FILE```. Where INPUT_FILE is the QA-SRL file, and the OUTPUT_FILE is where the Open IE file will be created. You can see that the script above just makes separate calls to this file, and then concatenates the outputs.

I'll add these instructions to the README, since I see that QA-SRL might arrive in different folder structures.
(*Update*: see https://github.com/gabrielStanovsky/oie-benchmark/blob/master/README.md#expected-folder-structure)
Please let me know if this works for you.

**Comment by gabrielStanovsky** [comment]
P.S. If you want to leave me your email 
(or send me a note at: ```gabriel (dot) satanovsky (at) gmail (dot) com```), i'll be happy to help you more closely with your specific setup.

**Comment by dahrs** [comment]
Hi @gabrielStanovsky,
Yes! Thanks. That's exactly the Answer I was expecting. I think I'll change my folder structure so it will match the one the benchmark expects. 
I saw the note on the QA-SRL site and I was hoping I wouldn't need the raw sentences since the CoNLL-2009 English training data is no longer available. I looked for it everywhere and even tried to run their python script (to complete their data with the raw sentences) with a newer version of CoNLL training data but it doesn't work. As expected.
So I think I'm once more in front of an impasse.
I'll mail you this same message, my mail address is sirchaos6 (at) gmail (d0t) com
